src/wmtk/simplex/cofaces_single_dimension.hpp

```diff
@@ -17,4 +18,10 @@ std::vector<Tuple> cofaces_single_dimension_tuples(
     const Mesh& mesh,
     const Simplex& my_simplex,
     PrimitiveType cofaces_type);
+
+std::vector<Simplex> cofaces_single_dimension_simplices(
```

Comment:
Getting the vector of simplices from the simplex collection is very simple. I don't think this needs its own function.